### PR TITLE
My Code Review Suggestions

### PR DESCRIPTION
**Pull request by pokhrebi**
### Description of the project
-- This is a reminder app that adds reminder based on location. When user is near added location, it will notify the user of the reminder. 

### Analysis of the program
-- Code was clean to read and go through. Project compiled with few warning and suggestions. Project was available on UC github on time. I think this project's UI is greatly written. I like how buttons are added and modified. 

* Removed extra unused lines
* Changed color to constant
* Added text color to make text visible

### What I learned from this code review
* I learned to add buttons and to modify the button
* I learned to add map
* I learned to add location on the map

### My Commits
* [cad3502f9c2d856572b960a9a43e59fc2e8052fb](https://github.com/Smoofington/FindMyRecyclingV32001/commit/cad3502f9c2d856572b960a9a43e59fc2e8052fb)
* https://github.com/Smoofington/FindMyRecyclingV32001/commit/e3f400a536e6d61f4fbdc4f1d9b612c6135980f5
* https://github.com/Smoofington/FindMyRecyclingV32001/commit/edf61e37ec6457681d82a06f70b19719964185a7

### CODE REVIEW

File: app/src/main/java/com/example/remindmelater/MainActivity.kt [hunk 1]
```diff
@@ -149,27 +152,26 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
...
-                    Text(text = "Map View")
+                    Text(text = "Map View", color = textColor)
                 }
             }
             Scaffold { innerPadding ->
                 LazyColumn(contentPadding = innerPadding) {
 
                 }
             }
-
```
Comment (pokhrebi) [inline app/src/main/java/com/example/remindmelater/MainActivity.kt:167]: removed extra unused line

File: app/src/main/java/com/example/remindmelater/MainActivity.kt [hunk 2]
```diff
@@ -192,16 +194,14 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
                     Text(text = "For:")
                 }
             }
-
```
Comment (pokhrebi) [inline app/src/main/java/com/example/remindmelater/MainActivity.kt:195]: removed extra unused line

File: app/src/main/java/com/example/remindmelater/MainActivity.kt [hunk 3]
```diff
@@ -192,16 +194,14 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
...
         }
-
```
Comment (pokhrebi) [inline app/src/main/java/com/example/remindmelater/MainActivity.kt:197]: removed extra unused line

File: app/src/main/java/com/example/remindmelater/MainActivity.kt [hunk 4]
```diff
@@ -89,14 +92,14 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
                         .padding(4.dp)
                         .width(190.dp)
                         .height(50.dp),
-                    colors = ButtonDefaults.buttonColors(backgroundColor = Color(12, 121, 230))
+                    colors = ButtonDefaults.buttonColors(backgroundColor = buttonColor)
```
Comment (pokhrebi) [inline app/src/main/java/com/example/remindmelater/MainActivity.kt:95]: DRY principle 

Added a hex code for same color in color resource, pulled that hex color in local variable and called it here

File: app/src/main/java/com/example/remindmelater/MainActivity.kt [hunk 5]
```diff
@@ -106,7 +109,7 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
...
```
Comment (pokhrebi) [inline app/src/main/java/com/example/remindmelater/MainActivity.kt:112]: DRY principle 

Added a hex code for same color in color resource, pulled that hex color in local variable and called it here

File: app/src/main/java/com/example/remindmelater/MainActivity.kt [hunk 6]
```diff
@@ -132,14 +135,14 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
                         .padding(4.dp)
                         .height(35.dp)
                         .width(190.dp),
-                    colors = ButtonDefaults.buttonColors(backgroundColor = Color(12, 121, 230))
+                    colors = ButtonDefaults.buttonColors(backgroundColor = buttonColor)
```
Comment (pokhrebi) [inline app/src/main/java/com/example/remindmelater/MainActivity.kt:138]: DRY principle 

Added a hex code for same color in color resource, pulled that hex color in local variable and called it here

File: app/src/main/java/com/example/remindmelater/MainActivity.kt [hunk 7]
```diff
@@ -149,27 +152,26 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
...
```
Comment (pokhrebi) [inline app/src/main/java/com/example/remindmelater/MainActivity.kt:155]: DRY principle 

Added a hex code for same color in color resource, pulled that hex color in local variable and called it here

File: app/src/main/java/com/example/remindmelater/MainActivity.kt [hunk 8]
```diff
@@ -45,24 +46,26 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
...
 
-
     @Composable
-    fun MainScreen(name: String) {
+    fun MainScreen() {
         val context = LocalContext.current
+        val greetingText = stringResource(R.string.greeting_text)
+        val buttonColor = colorResource(R.color.buttonColor)
+        val headerColor = colorResource(R.color.header_color)
+        val textColor = colorResource(R.color.white)
```
[the remark this answers is not in the document]
Comment (discospiff) [reply app/src/main/java/com/example/remindmelater/MainActivity.kt:61]: Go one step further, and move these to Colors.kt, which should hold standard colors for the look and feel.

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -1,4 +1,5 @@
 <resources>
     <string name="app_name">RemindMeLater</string>
     <string name="title_activity_maps">MapsActivity</string>
+    <string name="greeting_text">Hello, Set a Reminder for...</string>
```
Comment (discospiff) [inline app/src/main/res/values/strings.xml:4]: Good idea.

**Comment by discospiff** [comment]
Extracting string resources is a good idea.
Making constants out of frequently used colors is also a good idea.  Go one step further, and put them in Colors.kt, where colors belong.

File: app/src/main/java/com/example/remindmelater/MainActivity.kt [hunk 10]
```diff
@@ -36,7 +37,7 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
             RemindMeLaterTheme {
                 // A surface container using the 'background' color from the theme
                 Surface(color = MaterialTheme.colors.background) {
-                    MainScreen("Android")
+                    MainScreen()
                     Map()
```
Comment (discospiff) [inline app/src/main/java/com/example/remindmelater/MainActivity.kt:41]: duplicates previous review.

**Pull request closed without merging**